Services/calculator.js

```diff
@@ -0,0 +1,47 @@
+const {MONTHSINAYEAR, HUNDRED} = require("../Constants/index")
+
+//calculate the amount when delay is introduced
+const findAmountWithDelay = (options)=>{
```

Comment:
why to make two functions for just one change....change the logic if there is same work in both function try to do in one function us flag based work